st2common/bin/st2-run-pack-tests

```diff
@@ -146,19 +162,24 @@ else
         mkdir -p ${VIRTUALENVS_DIR}
         virtualenv --system-site-packages ${VIRTUALENV_DIR}
 
+        # Activate the virtualenv
+        source ${VIRTUALENV_DIR}/bin/activate
+        VIRTUALENV_ACTIVATED=true
+
         # Make sure virtualenv is using latest pip version
         ${VIRTUALENV_DIR}/bin/pip install --upgrade pip
     fi
 
-    # Activate created virtualenv
     if [ ! -d "${VIRTUALENV_DIR}" ]; then
         echo "Virtual environment ${VIRTUALENV_DIR} doesn't exist"
         echo "Run this script without the -j flag to create it or run it with -x flag to not use" \
              " a pack specific virtual environment."
         exit 2
     fi
 
+    # Activate created virtualenv
     source ${VIRTUALENV_DIR}/bin/activate
```

Comment:
Made a small change to use a function to activate it - 457d718bcedd646e91030bbb5932e9cef0ddbef0

As mentioned above, this might actually be and overkill and harder to read than the previous version (activate operating is idempotent so multiple activate calls don't break / hurt anything).